src/main/java/org/embulk/input/CommandFileInputPlugin.java

```diff
@@ -123,18 +122,17 @@ public TransactionalFileInput open(TaskSource taskSource, int taskIndex)
                 }
             }
         } catch (IOException ex) {
-            throw Throwables.propagate(ex);
+            throw new RuntimeException(ex);
         }
     }
 
-    @VisibleForTesting
-    static List<String> buildShell()
+    protected static List<String> buildShell()
```

Comment:
`protected` needed?

Reply:
Thanks I wanted to ask how to remove the `@VisibleForTesting` annotation. Fixed.